db/db.go

```diff
@@ -2273,6 +2273,25 @@ func GetTotalWithdrawals() (total uint64, err error) {
 	return
 }
 
+func GetWithdrawalsCountForQuery(query string) (uint64, error) {
+	bquery, _ := hex.DecodeString(strings.TrimPrefix(query, "0x"))
+	count := uint64(0)
+	err := ReaderDb.Get(&count, `
+			SELECT count(*)
```

Comment:
**issue:** I see this is essentially the same query as in GetWithdrawals but that query doesn't really work which is something that will be fixed in BIDS-2184. Whichever one gets merged first, the other one has to fix this too.

Reply:
applied the new logic also for counting